exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -536,6 +522,21 @@ void createTestKitWithTimeServiceAndTooManyValidators() {
     assertThat(thrownException).hasMessageContaining(expectedMessage);
   }
 
+  @Test
+  void getSnapshot(TestKit testKit) {
+    Snapshot view1 = testKit.getSnapshot();
+    Snapshot view2 = testKit.getSnapshot();
+    assertThat(testKit.snapshotCleaners).hasSize(2);
+
+    testKit.close();
+    Cleaner cleaner1 = view1.getCleaner();
```

Comment:
Snapshots have com.exonum.binding.core.storage.database.View.getViewNativeHandle
which is specified to throw ISE once it is closed. I'd use that instead of relying on
testkit internals.